### DISCUSSION TITLE
Which NuGet package? `Our.Umbraco.Community.Contentment` vs `Our.Umbraco.Community.Contentment.Core`

### DISCUSSION THREAD

**Discussion by pdao1** [leekelleher/umbraco-contentment#283] [Question] [answered]
Hi,

Came across both NuGet pages for Contentment, one Our.Umbraco.Community.Contentment and the other Our.Umbraco.Community.Contentment.Core

The one that's linked here (sans .Core) seemed to have less downloads vs the .Core one, and lacks the framework compatibility tab that .Core page has.  Can someone guide me on which one is the difference between the two?

Thank you

https://www.nuget.org/packages/Our.Umbraco.Community.Contentment
https://www.nuget.org/packages/Our.Umbraco.Community.Contentment.Core/#supportedframeworks-body-tab

**Comment by soreng** [comment, marked as answer]
https://www.nuget.org/packages/Our.Umbraco.Community.Contentment is the package you install in your Umbraco project, as it contains all the client side files.

https://www.nuget.org/packages/Our.Umbraco.Community.Contentment.Core contains all the domain stuff (models, services, contracts/interfaces, etc) that can be used in your own libraries to extend Contentment.

**Comment by leekelleher** [comment]
@pdao1 Thank you for the question. @soreng's answer is correct. 👍 _(Thanks Søren!)_

Coming to the project fresh, I can see how it'd be confusing which NuGet package to use.

Do you think there is anywhere in the documentation or README that could clarify this for new users?

**Reply by pdao1** [reply]
Thank you @soreng

@leekelleher Didn't see anything about Core on this readme, but I guess I just thought it was odd that the Core had more downloads, and it had all the compatible Dotnet framework tags while the primary one didn't.  So i had to double check XD

**Reply by leekelleher** [reply]
@pdao1 My general aim is to have people use the main Umbraco package plugin, (although I think I will amend the README file with details about the separate .Core library 👍).

The download count on the .Core library is crazy. Another NuGet package ([here](https://www.nuget.org/packages/N3O.Umbraco.Extensions/)) has a dependency on it, and that has some hefty download numbers. I can only assume that it's part of an agency's internal CI/CD build pipeline. I dunno, it's beyond my control. 🤷😅

**Comment by leekelleher** [comment]
FYI, since Contentment v5 (May 2023), the main NuGet package ID is: [`Umbraco.Community.Contentment`](https://www.nuget.org/packages/Umbraco.Community.Contentment/) (without the `Our.` prefix).
